Automattic/vip-go-mu-plugins
# Adding tracking for delete Parse.ly recommended widget

## Description

This PR adds a new event to the Parse.ly tracking module that tracks whenever a user has deleted the Parse.ly recommended widget. 

## Changelog Description

### Adding tracking for delete Parse.ly recommended widget

Tracking whenever the Parse.ly widget has been deleted.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Check out PR.
2. Make sure Parse.ly and Parse.ly tracking are active.
3. Create a Recommended Widget.
4. Delete the widget and check for the event on A8c Tracks.
